internal/signer/linux/util/util.go

```diff
@@ -47,10 +47,12 @@ func LoadConfig(configFilePath string) (config EnterpriseCertificateConfig, err
 	if err != nil {
 		return EnterpriseCertificateConfig{}, err
 	}
-	err = json.Unmarshal(byteValue, &config)
-	if err != nil {
-		return EnterpriseCertificateConfig{}, err
+
```

Comment:
nit: I think the old code style is a bit more idiomatic - since we are using a named return value, "config" should already be available in the scope to work with (no need to define a new ecpConfig). Btw, running "gofmt -w <filename>" is how you can locally format the go file for indentations etc. We should configure a linter for this repo at some point for more advanced code styling.